Makefile

```
@@ -48,4 +59,4 @@ test: deps
 
 full-test: deps
 	@echo ⚙️ Running all tests...
-	@cargo +nightly t -Fbinary-vendor -- --include-ignored --test-threads=1
+	@cargo +nightly t -Fbinary-vendor --workspace -- --include-ignored --test-threads=1
```

```suggestion
	@cargo +nightly t -Fbinary-vendor -- --include-ignored --test-threads=1
```
Why is this flag needed here?